core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -59,22 +55,8 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   // https://aphyr.com/posts/299-the-trouble-with-timestamps
   private val writeInProgress: JMap[String, Future[Done]] = new JHMap
 
-  // Announce written changes to DistributedPubSub if pubsub-minimum-interval is defined in config
-  private val pubsub = pubsubMinimumInterval match {
-    case interval: FiniteDuration =>
-      // PubSub will be ignored when clustering is unavailable
-      Try {
-        DistributedPubSub(context.system)
-      }.toOption flatMap { extension =>
-        if (extension.isTerminated)
-          None
-        else
-          Some(context.actorOf(PubSubThrottler.props(extension.mediator, interval)
-            .withDispatcher(context.props.dispatcher)))
-      }
-
-    case _ => None
-  }
+  // Is accessed from multiple threads via continuations on queries
+  private val tagActors = new ConcurrentHashMap[String, ActorRef]()
```

Comment:
Since the set of tags are typically static it might be slightly better performance to use AtomicRef with immutable Map, and CAS changes. Could be future optimization.

Reply:
but this will not be needed, see later...

Reply:
this should be an ordinary `Map`, not concurrent

Reply:
Yep, it was previously accessed via a call back :+1: